ci/goreleaser/goreleaser-fips.yml

```diff
@@ -0,0 +1,294 @@
+    - "--build-arg=GOLANG_CROSS={{ .Env.GOLANG_CROSS }}"
+  goarch: amd64
+  goos: linux
+  dockerfile: ci/images/plugin-compiler/Dockerfile
+  extra_files:
+    - ci/images/plugin-compiler
+    - go.mod
+    - apidef
+    - certs
+    - checkup
+    - cli
+    - config
+    - coprocess
+    - ctx
+    - dlpython
+    - dnscache
+    - gateway
+    - goplugin
+    - header
+    - log
+    - regexp
+    - request
+    - rpc
+    - signature_validator
+    - storage
+    - tcp
+    - templates
+    - test
+    - testdata
+    - trace
+    - user
+
+docker_manifests:
+  - name_template: tykio/tyk-gateway:{{ .Tag }}
+    image_templates:
+    - tykio/tyk-gateway:{{ .Tag }}-amd64
+    - tykio/tyk-gateway:{{ .Tag }}-arm64
+  - name_template: tykio/tyk-gateway:v{{ .Major }}.{{ .Minor }}{{.Prerelease}}
+    image_templates:
+    - tykio/tyk-gateway:{{ .Tag }}-amd64
+    - tykio/tyk-gateway:{{ .Tag }}-arm64
+  - name_template: tykio/tyk-hybrid-docker:{{ .Tag }}
+    image_templates:
+    - tykio/tyk-hybrid-docker:{{ .Tag }}-amd64
+    - tykio/tyk-hybrid-docker:{{ .Tag }}-arm64
+
+nfpms:
+  - id: std
+    vendor: "Tyk Technologies Ltd"
+    homepage: "https://tyk.io"
+    maintainer: "Tyk <info@tyk.io>"
+    description: Tyk API Gateway
+
+    package_name: tyk-gateway
+    builds:
+      - std-linux
+      - std-arm64
+    formats:
+      - deb
+      - rpm
+    contents:
+      - src: "README.md"
+        dst: "/opt/share/docs/tyk-gateway/README.md"
+      - src: "ci/install/*"
+        dst: "/opt/tyk-gateway/install"
+      - src: ci/install/inits/systemd/system/tyk-gateway.service
+        dst: /lib/systemd/system/tyk-gateway.service
+      - src: ci/install/inits/sysv/init.d/tyk-gateway
+        dst: /etc/init.d/tyk-gateway
+      - src: /opt/tyk-gateway
+        dst: /opt/tyk
+        type: "symlink"
+      - src: "LICENSE.md"
+        dst: "/opt/share/docs/tyk-gateway/LICENSE.md"
+      - src: "apps/app_sample.*"
+        dst: "/opt/tyk-gateway/apps"
+      - src: "templates/*.json"
+        dst: "/opt/tyk-gateway/templates"
+      - src: "templates/playground/*"
+        dst: "/opt/tyk-gateway/templates/playground"
+      - src: "middleware/*.js"
+        dst: "/opt/tyk-gateway/middleware"
+      - src: "event_handlers/sample/*.js"
+        dst: "/opt/tyk-gateway/event_handlers/sample"
+      - src: "policies/*.json"
+        dst: "/opt/tyk-gateway/policies"
+      - src: "coprocess/*"
+        dst: "/opt/tyk-gateway/coprocess"
+      - src: tyk.conf.example
+        dst: /opt/tyk-gateway/tyk.conf
+        type: "config|noreplace"
+    scripts:
+      preinstall: "ci/install/before_install.sh"
+      postinstall: "ci/install/post_install.sh"
+      postremove: "ci/install/post_remove.sh"
+    bindir: "/opt/tyk-gateway"
+    overrides:
+      rpm:
+        file_name_template: '{{ .PackageName }}-{{ replace .Version "-" "~" }}-1.{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}'
```

Comment:
Some new suffix will be needed here to distinguish these packages from the regular packages. Else they will have the same name and packagecloud will not allow them to be uploaded.